dist/releases/embeds.go

```diff
@@ -19,9 +24,11 @@ func GetVersions() []string {
 	if err != nil { // Should not happen.
 		return nil
 	}
-	names := make([]string, len(dirs))
-	for i, d := range dirs {
-		names[i] = d.Name()
+	names := []string{}
```

Comment:
I think you can still benefit from the optimization (that this commit removes) of knowing the slice length. You just use capacity rather than size for `make`.